pkg/scheduler/framework/plugins/nodelabel/node_label_test.go

```diff
@@ -244,3 +244,56 @@ func TestNodeLabelScore(t *testing.T) {
 		})
 	}
 }
+
+func TestNodeLabelFilterWithoutNode(t *testing.T) {
+	var pod *v1.Pod
+	tests := []struct {
+		name string
+		res  framework.Code
+	}{
+		{
+			name: "node does not exist",
+			res:  framework.Error,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			nodeInfo := schedulernodeinfo.NewNodeInfo()
+			p, err := New(nil, nil)
+			if err != nil {
+				t.Fatalf("Failed to create plugin: %v", err)
+			}
+			status := p.(framework.FilterPlugin).Filter(context.TODO(), nil, pod, nodeInfo)
+			if status.Code() != test.res {
+				t.Errorf("Status mismatch. got: %v, want: %v", status.Code(), test.res)
+			}
+		})
+	}
+}
```

Comment:
it is just a single test, and I don't expect other tests to be added here, please simplify the whole test to.

```
t.Run("node does not exist", func(t *testing.T) {
			nodeInfo := schedulernodeinfo.NewNodeInfo()
			p, err := New(nil, nil)
			if err != nil {
				t.Fatalf("Failed to create plugin: %v", err)
			}
			status := p.(framework.FilterPlugin).Filter(context.TODO(), nil, pod, nodeInfo)
			if status.Code() != framework.Error {
				t.Errorf("Status mismatch. got: %v, want: %v", status.Code(), test.res)
			}
		})
```

Reply:
Thanks, It has been done.